client/post-editor/editor-author/index.jsx

```diff
@@ -1,18 +1,24 @@
 /**
  * External dependencies
  */
-const React = require( 'react' );
+import React from 'react';
 
 /**
  * Internal dependencies
  */
-const Gravatar = require( 'components/gravatar' ),
-	user = require( 'lib/user' )(),
-	AuthorSelector = require( 'blocks/author-selector' ),
-	PostActions = require( 'lib/posts/actions' ),
-	touchDetect = require( 'lib/touch-detect' ),
-	sites = require( 'lib/sites-list' )(),
-	stats = require( 'lib/posts/stats' );
+import Gravatar from 'components/gravatar';
+import userFactory from 'lib/user';
+import AuthorSelector from 'blocks/author-selector';
+import PostActions from 'lib/posts/actions';
+import touchDetect from 'lib/touch-detect';
+import sitesFactory from 'lib/sites-list';
+import stats from 'lib/posts/stats';
```

Comment:
As you mentioned, we can repro the issue by navigating to a multi-author site, start a draft, then attempt to change author:
![screen shot 2016-11-16 at 4 29 34 pm](https://cloud.githubusercontent.com/assets/1270189/20371475/e1dcf28c-ac19-11e6-9245-b2ddec05aaad.png)